docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.memorybufferwindow.md

```diff
@@ -1,11 +1,12 @@
 ---
 #https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
-title: Window Buffer Memory (easiest)
-description: Documentation for the Window Buffer Memory node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+title: Window Buffer Memory (easiest) node documentation
```

Comment:
I think we can remove (easiest) from the title metadata (I think this was mostly to display in the side nav bar but since we're no longer using the title metadata, I don't see why we can't remove this) @freakwriter

Reply:
@ayatnkw thank you, this was one of the questions I had!